src/main/java/dev/codeninjas/book/service/service/BookServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package dev.codeninjas.book.service.service;
+
+import dev.codeninjas.book.service.ExceptionHandling.BookNotFoundException;
+import dev.codeninjas.book.service.domain.Book;
+import dev.codeninjas.book.service.repository.BookRepository;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@NoArgsConstructor(force = true)
```

Comment:
what is the purpose of force = true?
do we need it?

Reply:
it was forcing me add that or to add a constructor method